### DISCUSSION TITLE
dnsdist: spread notify among all backends

### DISCUSSION THREAD

**Discussion by SkySai1** [PowerDNS/pdns#14329] [Q&A] [answered]
Hello there!
Is anyone knows how to send notify query from primary server to all backends which is secondary servers via dnsdist?

I found [some instruction](https://dnsdist.org/advanced/axfr.html)s how to write politics but this not resolve the riddle

![spread notify](https://github.com/PowerDNS/pdns/assets/98015581/d90e395e-4ff4-4ffb-83d3-5db89da936ea)

**Comment by rgacogne** [comment, marked as answer]
Are you aware of `TeeAction`: https://dnsdist.org/reference/actions.html#TeeAction ? It can be used to duplicate `UDP` queries to a different IP:port than the selected backend, and since it does not stop the processing of subsequent rules, it can be used multiple times to duplicate the traffic to several hosts.

**Reply by SkySai1** [reply]
Yeap, ty dude, that was i need!
